modules/upgrade/pages/server-major-upgrade-uyuni.adoc

```diff
@@ -0,0 +1,102 @@
+[server-major-upgrade]
+= Server - Major Upgrade
+
+If {productname} core compontents are upgraded to new major versions, {productname} Server upgrades are considered as major.
+This is the case, if a version upgrade of PostgreSQL, Salt, or openSUSE Leap is needed.
+openSUSE Leap is the underlying base operating system (OS).
+For background information about upgrading openSUSE Leap, see the Startup Guide at https://doc.opensuse.org.
```

Comment:
```suggestion
For more information about upgrading openSUSE Leap, see the Startup Guide at https://doc.opensuse.org.
```

Reply:
I feel like this stuff about openSUSE Leap is a bit distracting. How relevant is it really?